packages/live-example/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@leafygreen-ui/live-example",
```

Comment:
if the live examples don't currently live with their packages, then do we even need to expose this as a package?